### PR TITLE
Add embed redirect toast

### PR DESCRIPTION
**Pull request by mhpjay422**
Why this PR?
- Adds a toast on redirect from embed

### CODE REVIEW

File: pages/embedded-portal.tsx [hunk 1]
```diff
@@ -292,38 +292,43 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 
   console.log("spaceData", spaceData);
 
+  console.log("getWorkbook race condition error occuring below. Resolving...");
```
Comment (mhpjay422) [inline pages/embedded-portal.tsx:295]: I can prevent the workbook error console log by wrapping async with the pattern below, but I would have to add a bunch of conditionals to the code

File: lib/hooks/usehooks.ts [hunk 2]
```diff
@@ -46,6 +46,9 @@ export function useFlashMessages(
   replaceUrl: string
 ) {
   useEffect(() => {
+    console.log("routerQuery", routerQuery);
+    console.log("replaceUrl", replaceUrl);
```
Comment (johnmosesman) [inline lib/hooks/usehooks.ts:50]: at least comment out

Comment (mhpjay422) [reply lib/hooks/usehooks.ts:50]: commented out

File: pages/embedded-portal.tsx (hunk 1 above)
Comment (mhpjay422) [reply pages/embedded-portal.tsx:295]: undid

File: pages/embedded-portal.tsx [hunk 3]
```diff
@@ -297,11 +297,10 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     workbookId: spaceData?.primaryWorkbookId!,
   });
 
-  console.log("workbook", workbook);
-
   const timeInFive = DateTime.now().plus({ seconds: 5 });
 
   while (!(workbook || DateTime.now() > timeInFive)) {
+    // This prevents the spamming of the console logs
```
Comment (johnmosesman) [inline pages/embedded-portal.tsx:303]: this comment needed?

**Review by johnmosesman — APPROVED** [review]
one comment, otherwise lgtm

File: pages/embedded-portal.tsx (hunk 3 above)
Comment (mhpjay422) [reply pages/embedded-portal.tsx]: removed

**Pull request merged**